google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts.
+After=local-fs.target network-online.target network.target
+After=google.service rsyslog.service
+Wants=local-fs.target network-online.target network.target
+
+[Service]
+ExecStart=/bin/true
```

Comment:
This didn't work when I tried it yesterday. It might work with a change to WantedBy like Andrey mentioned.